Explorer/Assets/DCL/MapRenderer/MapLayers/Pins/PinMarker.cs

```diff
@@ -107,12 +110,9 @@ public void SetTexture(Texture2D texture)
         public void OnBecameVisible()
         {
             poolableBehavior.OnBecameVisible();
-
-            if (poolableBehavior.instance == null) return;
-
-            if (Icon != null) { poolableBehavior.instance.SetTexture(Icon); }
-            poolableBehavior.instance.SetAsDestination(IsDestination);
-            poolableBehavior.instance.SetScale(currentNewScale);
+            if (Icon != null) { poolableBehavior.instance?.SetTexture(Icon); }
+            poolableBehavior.instance?.SetAsDestination(IsDestination);
+            poolableBehavior.instance?.SetScale(currentNewScale);
             ResetPulseAnimation();
```

Comment:
if poolableBehaviour is null the reset pulse animation might throw an error cause it access poolableBehavior

Reply:
well OnBecameVisible (which is called right before this) should get an instance into the poolableBehaviour, but I can add a check just in case :D